appveyor.yml

```diff
@@ -0,0 +1,29 @@
+version: "{build}"
+
+environment:
+  matrix:
+    - nodejs_version: 6
+    - nodejs_version: 8
+    - nodejs_version: 10
+
+matrix:
+  fast_finish: true
+
+platform: x64
+
+install:
+  - ps: Install-Product node $env:nodejs_version $env:platform
+  - npm install
+  - npm install typescript
+
+build: off
+
+test_script:
+  - .\node_modules\.bin\tsc --version
+  - .\node_modules\.bin\tsc -p test
```

Comment:
iirc `npx` is cross-platform. Either approach is 🆗with me.

Reply:
Could be, but `npx` was introduces with [`npm@5.2.0`](https://github.com/npm/npm/blob/latest/changelogs/CHANGELOG-5.md#v520-2017-07-05) so this won't work with [`node@6`](https://nodejs.org/en/download/releases/).

Reply:
Ah yes the "fun" of test matricies ^_^ ... technically we could do `npm i -g npm@5.2.0`, but it's a lot of time to add to each test run for little-to-no gain. Carry on 👍 🙏